app/views/podcast_player/_form.html.erb

```diff
@@ -0,0 +1,82 @@
+<%= turbo_frame_tag "embed-player", data: {controller: "morphdom"} do %>
+  <div class="col-12 mb-4">
+    <div class="form-floating input-group" data-controller="dynamic-form">
+      <%= select_tag :embed_player_type, embed_player_type_options(player_options[:embed_player_type]), class: "form-control", data: {action: "dynamic-form#change"} %>
+      <%= label_tag :embed_player_type, t("helpers.label.podcast_player.embed_player_type") %>
+      <a hidden href="<%= request.fullpath %>" data-dynamic-form-target="link"></a>
+    </div>
+
+    <div class="form-text">
+      <%= t(".help.types.#{player_options[:embed_player_type] || "standard"}") %>
+    </div>
+  </div>
+
+  <div class="col-12 mb-4">
+    <h3 class="fw-bold"><%= t(".title.playlist") %></h3>
+
+    <div class="form-text mb-4">
+      <%= t(".help.playlist") %>
+    </div>
+
+    <div data-controller="toggle-field">
+      <div class="form-floating input-group mb-2" data-controller="dynamic-form">
+        <%= select_tag :all_episodes, embed_player_all_episodes_options(player_options[:all_episodes]), class: "form-control", data: {action: "dynamic-form#change toggle-field#displayOnlyThisField"}, include_blank: false %>
+        <%= label_tag t("helpers.label.podcast_player.episode_number") %>
+        <%= field_help_text t(".help.episode_number") %>
+        <a hidden href="<%= request.fullpath %>" data-dynamic-form-target="link"></a>
+      </div>
+
+      <div class="<%= "d-none" unless player_options[:all_episodes] == "number" %> form-floating input-group mb-2" data-controller="dynamic-form" data-toggle-field-target="field" field="number">
+        <%= number_field_tag :episode_number, player_options[:episode_number], class: "form-control", data: {action: "dynamic-form#change"}, min: 1, autocomplete: "off" %>
+        <%= label_tag t("helpers.label.podcast_player.episode_number") %>
+        <a hidden href="<%= request.fullpath %>" data-dynamic-form-target="link"></a>
+      </div>
+    </div>
+
+    <div class="form-floating input-group mb-2" data-controller="dynamic-form">
+      <%= number_field_tag :season, player_options[:season], class: "form-control", data: {action: "dynamic-form#change"}, min: 0, autocomplete: "off" %>
```

Comment:
Realizing this and the other player page don't use the formbuilder (since there's no underlying model).

That does mean we're not getting the blank-field controller... I might at least add that here?  And start these fields out as `form-control-blank`.